editions/tw5.com/tiddlers/concepts/SystemTags.tid

```diff
@@ -12,6 +12,7 @@ These are the available system tags
 
 * {{$:/tags/AboveStory||$:/core/ui/TagTemplate}} for elements to be placed at the top of the story river
 * {{$:/tags/AdvancedSearch||$:/core/ui/TagTemplate}} for search elements
+* {{$:/tags/AdvancedSearch/FilterButton}} for filter buttons
```

Comment:
I don't understand why you're not using `{{$:/tags/AdvancedSearch/FilterButton||$:/core/ui/TagTemplate}}`?